features/in_onderzoek.feature

```diff
@@ -35,7 +35,7 @@ Functionaliteit: in onderzoek
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550001                       |
       | fields              | ouders,partners,<fields>        |
-      Dan heeft de persoon de volgende 'inOnderzoek' gegevens
+      Dan heeft de persoon de volgende 'persoonInOnderzoek' gegevens
       | naam                          | waarde   |
       | aanduidingGegevensInOnderzoek | <waarde  |
       | datumIngangOnderzoek          | 20120920 |
```

Comment:
Regels 42 en 43 horen hier n.m.m. niet in te staan aangezien naam.inOnderzoek en geboorte.inOnderzoek helemaal niet in de response van het gba post bericht voorkomen maar pas in de response die de proxy genereert. 
Ook begrijp ik niet wat regel 44 aan dit scenario toevoegt. Verblijfplaats wordt helemaal niet gevraagd dus het lijkt me logisch dat die niet wordt geleverd waarom moet je dat met regel 44 nog eens duiden.
Ik zal hiervoor een issue inschieten.